tests/vm_scheduling/env_config.yml

```diff
@@ -0,0 +1,8 @@
+env:
+  scenario: vm_scheduling
+  topology: azure.2019.10k
```

Comment:
use a toy topology instead. It's better to "debug". and better to be used for UnitTest here.

Reply:
reminder